frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/__snapshots__/CommandPalette.test.tsx.snap

```diff
@@ -5,13 +5,24 @@ exports[`snapshot test 1`] = `
   aria-describedby="radix-«r51»"
   aria-labelledby="radix-«r50»"
   class="_content_933568"
-  cmdk-dialog=""
   data-state="open"
   id="radix-«r4v»"
   role="dialog"
   style="pointer-events: auto;"
   tabindex="-1"
 >
+  <h2
+    hidden=""
+    id="radix-«r50»"
+  >
+    Command Palette
+  </h2>
+  <p
+    hidden=""
+    id="radix-«r51»"
+  >
+    A search-based interface that allows quick access to various commands and features within the application.
+  </p>
```

Comment:
The dialog title and description are currently outside of the cmdk-root,
but since they are within the dialog root, that should be fine.